qemu/tests/cfg/virtio_fs_share_data.cfg

```diff
@@ -106,22 +106,16 @@
         - @default:
         - @extra_parameters:
             variants:
-                - lock_posix_off:
-                    fs_binary_extra_options += " -o no_posix_lock"
```

Comment:
In fact, when one of the options is removed
There is only one combination left:
```
only Host_RHEL.m8
lock_posix_on,flock_on,xattr_on
```
Do we still need three variants?

Reply:
@YongxueHong What do you think?